packages/notifications/package.json

```diff
@@ -44,5 +44,5 @@
   },
   "zendeskgarden:library": "GardenNotifications",
   "zendeskgarden:max_size": "3 kB",
-  "zendeskgarden:src": "src/index.js"
+  "zendeskgarden:src": "src/index.ts"
```

Comment:
This should also include the `types` field

Reply:
Sounds good, I can add the `types` field. Should `"typings": ./index.d.ts`? That was [already in the file](https://github.com/zendeskgarden/react-components/blob/next/packages/notifications/package.json#L21).

Reply:
That `index.d.ts` can be removed. This was an attempt to manually include type definitions for some of our TS consumers.

You should be able to just have `"types": "dist/typings/index.d.ts",` like the other packages

Reply:
Great, I removed the file and updated the `package.json` 👍 .